google-datacatalog-apache-atlas-connector/setup.py

```diff
@@ -43,7 +43,8 @@
         'google-datacatalog-connectors-commons>=0.6.0'
     ),
     setup_requires=('pytest-runner',),
-    tests_require=('pytest-cov', 'google-datacatalog-connectors-commons-test>=0.6.0'),
+    tests_require=('numpy==1.19.4',
```

Comment:
WDYT about adding a single dependency per line and sorting them alphabetically?

Reply:
Sure about sorting them alphabetically, and I will check if putting each one in one line complies with yapf formatting.